content/docs/arvo/iris/tasks.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "API Reference"
+weight = 3
+template = "doc.html"
++++
+
+This document details the `task`s that can be sent to Iris. Iris only has a couple of `task`s you can use: [%request](#request) and [%cancel-request](#cancel-request).
```

Comment:
```suggestion
This document details the `task`s that can be sent to Iris. Iris only has a few `task`s besides the standard vane tasks: [%request](#request), [%cancel-request](#cancel-request), and [%receive](#receive).
```
Feel free to ignore this suggestion. I think you meant that there were only two tasks that you'd use from within Arvo, but my first reading made me think there were only two tasks at all, so I was then surprised when I saw a third one.

Reply:
I have rephrased this in https://github.com/urbit/urbit.org/pull/730/commits/4268a4ff7363cb065b4c0dece330dbf20e13ae7b